### DISCUSSION TITLE
Demos source not supported

### DISCUSSION THREAD

**Discussion by simonbusborg** [akiver/cs-demo-manager#748] [Q&A]
Does this even work? 
– I tried with 6 different demos now and I get the same message, even when changing source. How would you go about this?

![image](https://github.com/akiver/cs-demo-manager/assets/1101125/82dc211e-57d5-4090-bc9a-e5d4bc96b559)

**Comment by akiver** [comment]
Did you click and read the [documentation](https://cs-demo-manager.com/docs/faq/stats#the-demo-source-is-not-supported)?
If so, what don't you understand?

**Comment by simonbusborg** [comment]
I did, tried demos from different platforms, changing sources etc. still nothing. I wouldn’t have asked here if the documentation had solved my issue.On 4 Feb 2024, at 19.22, AkiVer ***@***.***> wrote:﻿
Did you click and read the documentation?
If so, what don't you understand?

**Reply by akiver** [reply]
Could you send a screenshot with the demo list and the selection you are trying to analyze?
Uploading the demos would also help.
